syncserver-settings/src/lib.rs

```diff
@@ -38,6 +38,8 @@ pub struct Settings {
     pub cors_allowed_methods: Option<Vec<String>>,
     pub cors_allowed_headers: Option<Vec<String>>,
 
+    pub actix_threadpool: Option<usize>,
```

Comment:
I think we should move away from this deprecated env var completely as actix-web did, it's confusing to keep it (call the setting something else).

I'm guess I'm not opposed to reading from the existing env var but TBH I'd rather just modify the cloudops config to set a new one alongside the old one and not even bother being backwards compatible, especially now that we have the time #killit